php/class-plugin.php

```diff
@@ -391,8 +391,12 @@ public function print_templates() {
 					<img src="{{ data.featured_image.sizes.thumbnail.url }}">
 					{{{ data.text }}}
 				</span>
-			<# } else { #>
+			<# } else if ( data.disable_edit ) { #>
```

Comment:
Would it be better to just make this `else if ( ! data.id )`, or identify why specifically there is no `id` present? We could then eliminate the `disable_edit` property.